### PR TITLE
Editorial

### PR DESCRIPTION
**Pull request by kaduk**
It's less overwhelming to do this in chunks as I make a review pass over the -18 for WGLC.
I may add more commits as I progress, but feel free to merge before I am done.

### CODE REVIEW

**Comment by ekr** [comment]
You'll probably want to make this build

**Comment by martinthomson** [comment]
It probably does build.  kramdown-rfc2629 has been getting consistent certificate errors from xml2rfc.tools.ietf.org.  I've told Henrik that he needs to add an intermediate.

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -1483,11 +1486,12 @@ Structure of this message:
            opaque legacy_session_id<0..32>;
            CipherSuite cipher_suites<2..2^16-2>;
            opaque legacy_compression_methods<1..2^8-1>;
-           Extension extensions<0..2^16-1>;
+           [[ Extension extensions<8..2^16-1>; ]]
```
Comment (davegarrett) [inline draft-ietf-tls-tls13.md]: I don't think inventing new syntax is a good idea here. Adding a comment on the line stating "OPTIONAL in old versions" would be better.

**Comment by martinthomson** [comment]
Build should now succeed if another commit is pushed (maybe one to remove the new notation Dave identified).

File: draft-ietf-tls-tls13.md (hunk 1 above)
Comment (martinthomson) [inline draft-ietf-tls-tls13.md]: Or we could take the 8 (is it only 8?) on the basis that we are describing what a TLS 1.3 client has to send, which always includes key_share or pre_shared_key and signature_algorithms.

**Comment by davegarrett** [comment]
On Thursday, November 10, 2016 10:34:02 am kaduk wrote:
> It's not new syntax; Section 3.2 notes that optional components are denoted by enclosing them in double brackets.

I stand corrected. It appears to only be used in the Presentation Language section. It's valid, but underused. If we're ok with not using it, I would still prefer to avoid it. Adding some note in a comment would be my preference vs. the current state.

**Comment by kaduk** [comment]
> It appears to only be used in the Presentation Language section

Right.  It occurred to me after I wrote the previous note, that perhaps the intent was to only
use the double-brackets-for-optional syntax when describing the presentation language itself (i.e., for optional struct tags), and not necessarily for structures defined using the presentation language.  Maybe @ekr has a better sense?

**Comment by ekr** [comment]
I merged most of this, and then did some more editorial work. Note in particular I didn't take the "compliant" piece because I don't think that makes sense in a spec context.

Yes, I read [[]] as being for the presentation language.

**Pull request closed without merging**